File: src/transformers/features/http/HttpIsFormDataTransformer.ts
import { HttpIsFormDataProgrammer } from "../../../programmers/http/HttpIsFormDataProgrammer";

import { GenericTransformer } from "../../internal/GenericTransformer";

export namespace HttpIsFormDataTransformer {
  export const transform = GenericTransformer.scalar("http.isFormData")(
    (project) => (modulo) => HttpIsFormDataProgrammer.write(project)(modulo),
  );
}

File: packages/transform/src/features/http/HttpIsFormDataTransformer.go
package http

import (
	corehttp "github.com/samchon/typia/packages/core/src/programmers/http"
	internal "github.com/samchon/typia/packages/transform/src/internal"
)

var HttpIsFormDataTransformer = httpIsFormDataTransformerNamespace{}

type httpIsFormDataTransformerNamespace struct{}

func (httpIsFormDataTransformerNamespace) Transform(call internal.ITypiaCallExpression) string {
	return corehttp.HttpIsFormDataProgrammer.Write(call.TypeText)
}
